### PR TITLE
Deploy to prod

### PR DESCRIPTION
**Pull request by arkid15r**
Production Deploy:
  * Project multiple contacts/publications support
  * Update Docker image
  * Take down ScPCA-nf recruitment banner

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -151,7 +152,7 @@ client/out/
 client/build
 
 # misc
-client/*.pem
+*.pem
```
Comment (nozomione) [inline .gitignore:155]: Move the misc to above the comment `# Client`.

Comment (arkid15r) [reply .gitignore:155]: Resolved in a Slack conversation.

Comment (nozomione) [reply .gitignore:155]: This will be adjusted in a separate PR, thus resolving this conversation.

**Review by nozomione — APPROVED** [review]
LGTM (this PR's files were already approved in previous PRs)

**Pull request merged**